script/run-cypress-tests-docker.sh

```diff
@@ -2,5 +2,5 @@ if [ "$(find src -name '*.cypress.spec.js' | wc -l)" -eq 0 ]; then
   echo "No Cypress tests found."
   exit 0
 else
-  yarn cy:run --config baseUrl=http://vets-website:3001
+  CYPRESS_BASE_URL=http://vets-website:3001 yarn cy:run
```

Comment:
Not much of a diifference (other than a minor one in how configs take precedence).

This was just left over from when I was passing other `--config` options specifically for `cy:run`, which would get overwritten by the following `--config`. But I kind of like that this reads more clearly (and loudly).